database/rrdhost.c

```diff
@@ -710,6 +710,18 @@ void rrdhost_save_charts(RRDHOST *host) {
     rrdhost_unlock(host);
 }
 
+static int is_valid_label_value(char *value) {
+    while(*value) {
+        if(*value == '"' || *value == '\'') {
```

Comment:
This does not match the docs - should exclude `*` and `!` as well.

Reply:
Thank you, I changed the function!